From 4469dbbe84e01c611d8a3a02ee992a6ee8640d39 Mon Sep 17 00:00:00 2001
From: Martin Finkel <martin@videolabs.io>
Date: Mon, 6 Apr 2020 13:30:43 +0200
Subject: [PATCH 1/1] Patch Linux csproj to work with .NET Core 3

---
 src/LibVLCSharp/LibVLCSharp.csproj | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/LibVLCSharp/LibVLCSharp.csproj b/src/LibVLCSharp/LibVLCSharp.csproj
index f896126..e787981 100644
--- a/src/LibVLCSharp/LibVLCSharp.csproj
+++ b/src/LibVLCSharp/LibVLCSharp.csproj
@@ -1,4 +1,4 @@
-﻿<Project Sdk="MSBuild.Sdk.Extras">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Title>LibVLCSharp</Title>
     <Summary>The official .NET wrapper around LibVLC.</Summary>
-- 
2.17.1

